### PR TITLE
minor cleanup and name change from gquorum -> goquorum

### PR DESCRIPTION
**Pull request by joshuafernandes**
- Ver bump to release
- Change 'gquorum' -> 'goquorum' as per product name and docs etc
- Minor reverts to tx example code to keep it consistent with docs & goquorum examples

### CODE REVIEW

File: files/besu/smart_contracts/scripts/private_tx.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const contractJson = JSON.parse(fs.readFileSync(contractJsonPath));
 const bytecode = contractJson.evm.bytecode.object
 const contractAbi = contractJson.abi;
 // initialize the default constructor with a value `47 = 0x2F`; this value is appended to the bytecode
-const contractConstructorInit = "0000000000000000000000000000000000000000000000000000000000001337";
+const contractConstructorInit = "000000000000000000000000000000000000000000000000000000000000002F";
```
Comment (joshuafernandes) [inline files/besu/smart_contracts/scripts/private_tx.js:16]: Reverting this change to keep it consistent with docs & with goQuorum examples

File: files/besu/smart_contracts/scripts/private_tx.js [hunk 2]
```diff
@@ -86,7 +86,7 @@ async function main(){
   createContract(besu.member1.url, besu.member1.privateKey, tessera.member1.publicKey, tessera.member3.publicKey)
   .then( async function(privateTxReceipt){
     console.log("Address of transaction: ", privateTxReceipt.contractAddress);
-    let newValue = 357;
+    let newValue = 123;
```
Comment (joshuafernandes) [inline files/besu/smart_contracts/scripts/private_tx.js:89]: Reverting this change to keep it consistent with docs & with goQuorum examples

File: files/besu/smart_contracts/scripts/public_tx.js [hunk 3]
```diff
@@ -66,7 +66,7 @@ async function createContract(host, contractBin, contractInit) {
 };
 
 async function main(){
-  let newValue = 789;
+  let newValue = 123;
```
Comment (joshuafernandes) [inline files/besu/smart_contracts/scripts/public_tx.js:69]: Reverting this change to keep it consistent with docs & with goQuorum examples

File: files/orchestrate/README.md [hunk 4]
```diff
@@ -362,7 +362,7 @@ PegaSys Orchestrate is compatible with **Quorum Tessera** for private transactio
 
 <!-- Then following instructions: -->
 <!-- - [Hyperledger Besu](../besu/README.md) -->
-<!-- - [GoQuorum](../gquorum/README.md) -->
+<!-- - [GoQuorum](../goquorum/README.md) -->
```
Comment (joshuafernandes) [inline files/orchestrate/README.md:366]: Don't know why it wasn't called `goquorum` from the beginning

**Review by ricardolyn — APPROVED** [review]
👍

**Pull request merged**